ui-meta/workstation.yml

```diff
@@ -47,6 +47,11 @@ usage_template: |-
 
   Monitoring dashboards for the workstation are available using the [Monitoring service]({{ monitoring.url }}).
 
+  This Azimuth user is mounted at {{ user_mountpoint }}/{{ azimuth_username }}-home
+
+  Note that this user storage is EPHEMERAL and is removed on platform deletion, use
+  /platform for persistent storage between workstations.
```

Comment:
This needs revising, lets review slurm. We need to cover:

- azimuth user home is deleted when the platform is deleted
- ... but is persisted across patch. Although all other system modifications are lost on patch.